go-controller/pkg/clustermanager/userdefinednetwork/controller_test.go

```diff
@@ -1353,14 +1470,9 @@ func testNADWithDeletionTimestamp(ts time.Time) *netv1.NetworkAttachmentDefiniti
 }
 
 func testNamespace(name string) *corev1.Namespace {
-	return &corev1.Namespace{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: name,
-			Labels: map[string]string{
-				"kubernetes.io/metadata.name": name,
-			},
-		},
-	}
+	ns := invalidTestNamespace(name)
+	ns.ObjectMeta.Labels[ovntypes.RequiredUDNNamespaceLabel] = ""
+	return ns
```

Comment:
nit: I don't see a test for case3 where pod gets attached to default network.. maybe its in other files.. but good to track that behaviour somewhere...

Reply:
these tests are just in clustermanager, so there is no test for pods here. I'll add a test in the  ovn/secondary_layer2/3 files to cover a UDN exists, but namespace not labeled, and see pod is attached to default network.